README.md

```diff
@@ -51,6 +51,8 @@ It is recommended to use this extension with [phpstan-nette](https://github.com/
 * [Link checking](docs/configuration.md#link-checking)
 * [Other configuration options](docs/configuration.md#other-configuration-options)
 
+If analysis is slow you should set option `latte.collectedPaths` as desciption in [Analyser configuration](docs/configuration.md#analyser-configuration).
```

Comment:
- if you need to collect (not analyse - no errors will be reported) extra files or directories 
- typo description